src/main/java/com/purbon/kafka/topology/CommandLineInterface.java

```diff
@@ -120,6 +132,7 @@ private Options buildOptions() {
     options.addOption(brokersListOption);
     options.addOption(adminClientConfigFileOption);
 
+    options.addOption(overridingAdminClientConfigFileOption);
```

Comment:
i know is a small thing, but ... we could probably name these variables more clearly ... something like _ overridingConfigFileOption_ 

It is not your thing, you did the right thing, however because of history this started only being adminClient, but now it does much more ... this is why I say. Makes sense?

Reply:
this we can do for the override and as well for the main config. What do you think?